### PR TITLE
Set up auto

### LINKED ISSUE #132: auto-mate releases
I got really spoiled by `auto` ;)

### PR DESCRIPTION
**Pull request by jwodder**
Closes #132.

This PR also needs to be accompanied by the following changes:
- [x] Create the necessary labels for `auto` by running `GH_TOKEN=... auto create-labels` in a copy of this repository containing `.autorc`
- [x] All PRs merged since the previous release should receive appropriate labels
- [x] An auth token for uploading assets for the `pyout` project to PyPI must be saved as a secret named "`PYPI_TOKEN`"

### CODE REVIEW

**Comment by yarikoptic** [comment]
invited @jwodder to pypi for pyout

**Comment by jwodder** [comment]
@yarikoptic PyPI token created & saved.

File: .autorc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "onlyPublishWithReleaseLabel": true,
+    "baseBranch": "master",
+    "author": "auto <auto@nil>",
+    "noVersionPrefix": false,
+    "plugins": [
+        "git-tag",
+        [
+            "exec",
+            {
+                "afterChangelog": "bump2version \"$(printf '%s\n' \"$ARG_0\" | jq -r .bump)\"",
```
Comment (yarikoptic) [inline .autorc:11]: could you please remind me why we need the bump2version?

Comment (yarikoptic) [reply .autorc:11]: ah, because of no versioneer or alike used here, right?

Comment (jwodder) [reply .autorc:11]: Correct.  pyout sets its version directly in `setup.py` and `pyout/__init__.py`, so we use bump2version to increment it.

**Comment by yarikoptic** [comment]
Let's proceed
- could you please rebase on current master so CI gets green
- can we mark this one for `release` itself? (worth kicking release due to jsonschema fixes)

**Comment by jwodder** [comment]
@yarikoptic Rebased.  I believe marking this as "release" should work; done.

**Comment by yarikoptic** [comment]
awesome, thank you @jwodder ! let's proceed!

**Pull request merged**